configure

```diff
@@ -89,6 +89,12 @@ mynormalizepath () {
       ;;
   esac
 
+  if [ ! -d $P ]
+  then
+    echo "Prefix directory does not exist, creating directory."
+    mkdir $P
```

Comment:
Maybe it should use `mkdir -p` to construct intervening paths as well?

Reply:
done